buildkite/culprit_finder.py

```diff
@@ -59,8 +59,14 @@ def get_bazel_commits_between(first_commit, second_commit):
         )
 
 
+def get_platform(project_name, task_name):
+    http_config = DOWNSTREAM_PROJECTS[project_name]["http_config"]
```

Comment:
This has already been existing before, but can we only import bazelci and then refer to its fields as bazelci.DOWNSTREAM_PROJECTS and bazelci.PLATFORMS?

Reply:
No problem!